backend/organization/views/project_views.py

```diff
@@ -1066,6 +1064,33 @@ def get_queryset(self):
         return open_membership_requests
 
 
+class HasUserRequested(APIView):
```

Comment:
Nit: Can you change the class name here? It is confusing to the reviewer. What does `HasUserRequested` means? It can mean multiple things. We should name this class based on its usage.